### ISSUE TITLE
Re-enable caching for /about/experts-staff

### ISSUE THREAD

**Issue by mariacha** [wri/wri_sites#227]
When we added metatags for different search_api_solr pages, we lost views caching. This ticket would re-enable it for the resources listings, experts, and events pages on all sites. They're already re-enabled on Flagship.

**Comment by StudioZut** [comment]
@mariacha is there more work for this? If it's enabled on Live then is it complete, or do you need to pull the config back into the profile?

**Comment by mariacha** [comment]
Yeah, I just need to review if they're enabled on all the multidevs and in the profile. I know China doesn't use the page anyway.
